resources/js/Pages/Layout.vue

```diff
@@ -190,15 +190,16 @@ onBeforeUnmount(() => {
 <style lang="scss">
 @import "@wikimedia/codex-design-tokens/theme-wikimedia-ui";
 @import '../../sass/typography';
+@import '../../css/custom-variables.css';
 
 #app {
     box-sizing: border-box;
     min-height: 100%;
     display: flex;
     flex-direction: column;
-
+    
```

Comment:
:grimacing:

Reply:
There's this very cool plugin you can install that will get rid of that white space: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig